captum/attr/_core/lrp.py

```diff
@@ -408,6 +408,7 @@ def _forward_fn_wrapper(self, *inputs: Tensor) -> Tensor:
 
 
 SUPPORTED_LAYERS_WITH_RULES = {
+    nn.Identity: lambda: EpsilonRule(epsilon=0.0),
```

Comment:
Thank you for  the PR, @soupault! I have  a question regarding epsilon=0.0.

 It can be that the output of the identity is 0  and we might hit division by zero ?

https://github.com/pytorch/captum/blob/7c08d06428f867e0a487c092b06f06514c498cbe/captum/attr/_utils/lrp_rules.py#L64

Reply:
If we want no-op for Identity we could potentially add it a list like `SUPPORTED_NON_LINEAR_LAYERS ` ?

Reply:
Thank you @NarineK for the review! Division by 0 is definitely possible, yet I would say rather unrealistic when Identity is used with any other layer and real data. In any case, better to set the rule to the common Epsilon. No-op is surely the best fit here, but I feel it might be confusing for further contributors to have the layer under `_NON_LINEAR_`.